ionic.config.json

```diff
@@ -1,8 +1,7 @@
 {
   "name": "PVTrAck",
-  "app_id": "",
-  "type": "ionic-angular",
   "integrations": {
     "cordova": {}
-  }
-}
+  },
+  "type": "ionic-angular"
```

Comment:
What does putting it here do?

Reply:
It was a fix that was made when I ran a command.